python/bids_import.py

```diff
@@ -216,6 +216,20 @@ def read_and_insert_bids(bids_dir, config_file, verbose, createcand, createvisit
                     loris_bids_root_dir    = loris_bids_root_dir
                 )
 
+            elif modality == 'ieeg':
+                Ieeg(
```

Comment:
I compared the `Ieeg` and `Eeg` classes and I did not notice any major code changes (aside from renaming variables and documentation to `Ieeg` instead of `eeg`).

Therefore, I don't see the need to create a new `Ieeg` class when it is identical to the `Eeg` class. Instead, I would just call the `Eeg` class here.

If there are some specificities I missed for `Ieeg`, then, this could be resolved with if statements in the class and checking the modality in the `Eeg` class.